XenAdmin/ConsoleView/XSVNCScreen.cs

```diff
@@ -686,10 +686,15 @@ private bool InDefaultConsole()
         private void startPolling()
         {
             //Disable the button first, but only if in text/default console (to allow user to return to the text console - ref. CA-70314)
-            if (InDefaultConsole())
-            { 
+            if (Helpers.CreedenceOrGreater(Source.Connection))
+            {
+                parentVNCTabView.EnableToggleVNCButton();
+            }
+            else if (InDefaultConsole())
+            {
                 parentVNCTabView.DisableToggleVNCButton();
             }
+            
 
             //Start the polling again

```

Comment:
The design asks for polling not to happen for Cream and later servers. Now that you have modified the guest agent to notice changes to RDP, we should never need to poll.